tests/tests.py

```diff
@@ -1450,6 +1450,8 @@ def _retval_setup(self, cur):
 CREATE PROCEDURE add_one (@input int)
 AS
 BEGIN
+    select 'a' as a
+    select 'b' as b
```

Comment:
Why did you change this test case?  I think it was useful, it tested stored procedures without any result-sets but with return values.